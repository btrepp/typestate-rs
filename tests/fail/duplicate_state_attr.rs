use typestate_proc_macro::typestate;

#[typestate]
mod m {
    #[automaton]
    struct S {}

    #[state]
    #[state]
    struct A {}
}

fn main() {}