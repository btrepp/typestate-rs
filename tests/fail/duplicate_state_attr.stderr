error: Duplicate attribute.
 --> $DIR/duplicate_state_attr.rs:9:5
  |
9 |     #[state]
  |     ^^^^^^^^
